app/serializers/journey_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class JourneySerializer < ActiveModel::Serializer
+  attributes :id, :state, :billable, :vehicle
+  attribute :client_timestamp, key: :timestamp
+
+  has_one :from_location, serializer: LocationSerializer
```

Comment:
It doesn't seem to be necessary to explicitly state the serializer, this works just as well (see `AllocationSerializer` for example):

```ruby
has_one :from_location
has_one :to_location
```

Reply:
thanks, now fixed